scripts/brackets.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+start Brackets.exe %*
```

Comment:
I guess what I'm wondering is, what are the differences between doing that vs. just doing `..\Brackets.exe %*` instead?